integration/ui/BUILD

```diff
@@ -1,11 +1,10 @@
-load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
-load("@npm//@bazel/typescript:index.bzl", "ts_library")
+load("@com_github_airyhq_bazel_tools//web:typescript.bzl", "ts_web_library")
 
-ts_library(
+ts_web_library(
     name = "specs",
     srcs = glob(["*.spec.ts"]),
-    tsconfig = ":tsconfig.json",
     deps = [
+        "//frontend/ui:app",
```

Comment:
This should not depend on the app target but on the `:handler` but then the build fails. I think because the handler.js is picked up by both rules. Not sure how to fix that. Please help

Reply:
So the idea was to do a separate `ts_library` only for the labels that gets used as an import in `//frontend/ui:app` and here.

Reply:
yes! I had this but I got a weird error that the handles.js is targeted twice from the :app and the :handles target
https://github.com/airyhq/airy/pull/1087/commits/dd87b9be16e460188de45118fd18ab4c834d8217

Reply:
Yes, then you have to exlcude the `handles.ts` in the source glob pattern of the `:app` target :)

Reply:
Or move it to a completely different module